release-notes/Release-Notes-RC2.md

```diff
@@ -0,0 +1,144 @@
+* Server GC on Unix platforms
+
+### JIT
+
+* Port of all JIT changes made for .NET Framework 4.6.1
+* SIMD code generation for System.Numerics on all platforms
+* Code quality improvements
+
+### BCL
+
+Below is a summary of API changes made since RC1. A complete and detailed listing of API changes can be seen [here](RC1-RC2_API_diff.md).
+
+New .NET Core APIs
+
+* System.Drawing
+* System.IO.BufferedStream
+* System.IO.Packaging
+
+APIs added in RC2 to existing classes and namespaces
+
+* System.Console
+* System.Diagnostics.ProcessStartInfo
+* System.Environment
+* System.Linq
+* System.Reflection
+* System.Runtime.InteropServices
+* System.Runtime.InteropServices.RuntimeInformation
+* System.Security.Cryptography (additional Crypto details in the next section)
+* System.Security.Principal
+* System.Security.WindowsPrincipal
+* System.ServiceModel
+
+### Cryptography
+
+* RSA.Create() was added to support platform-agnostic operations
+* TripleDES (3DES) was added (it existed, but threw NotImplementedException, in RC1)
+* Reference Assemblies: HMACMD5 was missing in RC1, added it
+* Reference Assemblies: Refactored ECDSA into a higher version to allow net46-compatible references for all cryptography packages
+* Unix: X509Chain.ChainStatus and X509Chain.ChainElementStatus now report all chain errors, not just the first one
+* Unix: Finish X509Certificates support
+    * Implement GetCertContentType
+    * Constructors and properties for all X509Extension classes behave as on Windows
+    * Add support for X509ContentType.Pkcs7 for X509Certificate2Collection.Export
+    * Add support for X509Chain.ChainPolicy.VerificationFlags for error suppression
+    * Breaking change: X509Certificate2.FriendlyName and X509Certificate2.Archived setters throw PlatformNotSupportedException
+* Unix: The CA cert bundle read into the machine Root CA store now also supports the one unified file format
+* Unix: Default RSA keysize is 2048. This matches Windows .NET Core, but not Windows .NET Framework (1024-bit default)
+* OS X: Minimum version of OpenSSL bumped from 0.9.8 to the 1.0.0 family
+    * The on-box libcrypto.0.9.8.dylib has gone out of security-fix support, libcrypto.1.0.0.dylib must be installed via homebrew
+* Windows: An ECDSA PFX created with OpenSSL can now be successfully used
+* Windows: RSA public and private keys with unusual sizes (e.g. 1032-bit) work now
+
+### Serialization
+
+* ISerializatonSurrogateProvider interface now supports applications or libraries that target for either full .NET Framework or .NET Core to use data contract surrogate in DataContractSerializer.
+* DataContractJsonSerializer improved compliance with ECMAScript 6 standard in serializing control characters including 0x0008 (BACKSPACE), 0x000C (FORM FEED), 0x000A (LINE FEED), 0x000D (CARRIAGE RETURN), and 0x0009 (HORIZONTAL TABULATION).
+
+Additional details can be found in the [WCF Release Notes](https://github.com/dotnet/wcf/releases/tag/v1.0.0-rc2)
```

Comment:
@leecow can we have WCF called out as a separate bullet point for better discovery? The WCF release note does not really provide additional details for serialization. Thanks!